framework/src/modules/interoperability/mainchain/cc_commands/channel_terminated.ts

```diff
@@ -12,17 +12,23 @@
  * Removal or modification of this copyright notice is prohibited.
  */
 
-import { BaseCCCommand } from '../../base_cc_command';
+import { BaseInteroperabilityCCCommand } from '../../base_interoperability_cc_commands';
 import { CROSS_CHAIN_COMMAND_ID_CHANNEL_TERMINATED } from '../../constants';
 import { channelTerminatedCCMParamsSchema } from '../../schema';
+import { CCCommandExecuteContext, StoreCallback } from '../../types';
+import { MainchainInteroperabilityStore } from '../store';
 
-export class CCChannelTerminatedCommand extends BaseCCCommand {
+export class CCChannelTerminatedCommand extends BaseInteroperabilityCCCommand {
```

Comment:
maybe better to have a prefix `MainchainCCChannelTerminatedCommand` and same for Sidechain and its commands as there will be 2 different classes with the same name

Reply:
updated